readme.md

```diff
@@ -17,221 +16,31 @@ MISO: An open source LIMS for small-to-large scale sequencing centres
-then it is unlikely that you will need to change any options, but the LDAP setup is far more complex.
-Please ask your sysadmin to match up the MISO LDAP options with your local LDAP server options.
-
-Secondly, the `security.method` environment property relates to the security mechanism, i.e. 'ldap' or 'jdbc', where 'ldap' auths
-against an LDAP server, and 'jdbc' auths against a local database (usually the MISO database itself). Please use 'jdbc' if you
-are unsure. There is a default admin user that is shipped with the MISO database, and the username/password login is admin/admin.
-
-To set the `security.method` property, use local environment variables on Tomcat startup. Please see the "Starting MISO" section below.
-
-> NB: You are likely to see startup errors like the one below if you do not set this property, and start Tomcat as outlined in section 3!
-
-> 20-Oct-2010 11:08:28 org.apache.catalina.core.StandardContext start
-
-> SEVERE: Error listenerStart
-
-> 20-Oct-2010 11:08:28 org.apache.catalina.core.StandardContext start
-
-> SEVERE: Context [/tgac] startup failed due to previous errors
-
-4.5 ) Submission properties
 
-MISO is able to submit sequence data to the major sequence archives, i.e. the SRA at the EBI, and the EMBL GenBank. Currently,
-only ERA submissions are supported out-of-the-box. The settings for these services are specified in the `submission.properties` file.
+View [Full MISO Documentation](https://documentation.tgac.ac.uk/display/MISO/MISO+Documentation+Home)
 
-5 ) STARTING MISO
------------------
+## Running an Instance of MISO
+To run your own MISO instance in the long term, you will need to maintain your
+own fork of the MISO codebase with your configuration changes. For each
+release, you will need to merge the main changes into your local repository and
+then deploy a new version.
 
-To set the `security.method` property to use the local MISO database and start the Tomcat instance, run the following command, which will use the JDBC security.method and assign 768MB RAM to the Java heap:
+For deployment, you will need a mySQL database server, a server to scan the
+sequencer output directories (the notification server), and a Tomcat
+application server. These can be on the same machine. It is probably useful to
+have a separate build environment.
 
-    cd <tomcat-install>/bin/
-    JAVA_OPTS="$JAVA_OPTS -Dsecurity.method=jdbc -Xmx768M" ./startup.sh
+At each upgrade, there are two steps: migrating the database and deploying a
+new application. Each version of the application contains all the migrations
+needed to upgrade any old database to match the current version. However, there
+is no rollback procedure, so a full database backup should be taken before
+migration.
 
-All going well, you should be able to go to http://localhost:8080/ and log in to MISO using the admin/admin username and password combination.
+The details can be found in the [MISO maintainer guide](admin-manual.md).
```

Comment:
Installation and configuration details can be found in the [MISO maintainer guide](admin-manual.md).